app/views/devise/registrations/new.html.erb

```diff
@@ -13,12 +13,12 @@
     <p><%= t("authentication.mandatory_warning") %></p>
     <%= f.validation_summary %>
 
-    <div class="registration__row">
-      <div class="registration__field">
+    <div class='registration__row'>
+      <div class='registration__field', id='first_name'>
```

Comment:
@asmega the idea was added before I realised that there was a gem called SitePrism which, was capable of picking up this element without having to explicitly set an id. 

I will remove this as well as fix the comma in the HTML attributes